pkgs/top-level/all-packages.nix

```diff
@@ -10746,6 +10746,8 @@ with pkgs;
 
   tinystatus = callPackage ../tools/networking/tinystatus { };
 
+  toastify = callPackage ../tools/misc/toastify {};
```

Comment:
```suggestion
  toastify = darwin.apple_sdk_11_0.callPackage ../tools/misc/toastify {};
```

Reply:
had to do this manually, it kept telling me the diff changed on webui even after refreshing